src/components/outline/Display.tsx

```diff
@@ -169,15 +178,20 @@ const Display: React.FC = () => {
           </Box>
         </Flex>
         <br></br>
-        <Box paddingLeft={"5%"} paddingRight={"5%"}>
-          <Text fontSize={32}>{title}</Text>
-          <br></br>
-          <Flex flexWrap="wrap" justifyContent="space-evenly">
-            {data?.hexathonUsers
-              .filter((hUser: any) => hUser.userId !== user?.uid)
-              .map((user: UserCardType) => <UserCard key={user.name} {...user} />)}
-          </Flex>
+        <Box display="flex" justifyContent="space-between" alignItems="center" borderWidth="1px" borderRadius="lg" borderColor="purple" width="200px" marginLeft={"auto"} marginRight={"auto"}>
+            <Button colorScheme='purple' variant={displayMode=="allUsers" ? "solid": "ghost"} width="100px" onClick={displayUsers}>
+                Individuals
+            </Button>
+            <Button colorScheme='purple' variant={displayMode=="allTeams" ? "solid": "ghost"} width="100px" onClick={displayTeams}>
+                Teams
+            </Button>
         </Box>
+        {displayMode=="allUsers" && (
+          <UserDisplay data={data}></UserDisplay>
+        )}
+        {displayMode=="allTeams" && (
+          <TeamsDisplay></TeamsDisplay>
+        )}
```

Comment:
For components that do not wrap other components, you can just self close them like so `<UserDisplay data={data} />`. Change it for teams display as well below